src/test/ui/consts/const-eval/const_raw_ptr_ops.stderr

```diff
@@ -1,18 +1,49 @@
-error: pointers cannot be reliably compared during const eval
```

Comment:
The const checking logic for this message can now just be an assert or a delay_span_bug I believe